configs/10-stashcache-auth-overrides.conf

```diff
@@ -0,0 +1,14 @@
+
+[Unit]
+
+# Service dependencies:
+# - Hard dependency on the proxy generation: wait until done and require it
+#   to succeed.
+# - Make sure the authfile is generated before startup, but make it a softer
+#   dependency: we want to allow Xrootd to restart even if the topology service
+#   is down.
+# - Depends on fetch-crl and the reporter script, but doesn't need these
+#   to finish or succeed for startup.
+Requires=xrootd-renew-proxy.service
+Wants=xrootd-renew-proxy.timer stashcache-reporter.service stashcache-reporter.timer fetch-crl-boot.service fetch-crl-cron.service stashcache-authfile.service stashcache-authfile.timer
```

Comment:
Do you have to list both the service and timer for services that have both?  All you want is the timer to be activated, which in turn will activate the service.  Having the service here, I wonder if it will just start it once, and it will die quickly (one-shot?)

Reply:
Fixed.